cypress/e2e/tests/pages/cluster-manager.spec.ts

```diff
@@ -72,6 +73,25 @@ describe('Cluster Manager', { tags: '@adminUser' }, () => {
         detailRKE2ClusterPage.waitForPage(undefined, 'registration');
       });
 
+      it('can copy config to clipboard', () => {
+        ClusterManagerListPagePo.navTo();
+
+        cy.intercept('POST', '*action=generateKubeconfig').as('copyKubeConfig');
+        clusterList.list().actionMenu(rke2CustomName).getMenuItem('Copy KubeConfig to Clipboard').click();
+        cy.wait('@copyKubeConfig');
+
+        // Verify confirmation message displays and is hidden after ~3 sec
+        cy.get('.growl-text').contains('Copied KubeConfig to Clipboard').should('be.visible');
```

Comment:
Comment only.

At some point we should make a growl PO. If we use it more we should do so, but at the moment this is fine.